### DISCUSSION TITLE
Courseplay window in wide resolution on three monitors 6060x1080

### DISCUSSION THREAD

**Discussion by BanShieRU** [Courseplay/Courseplay_FS22#2817] [Q&A]
When using three monitors and a resolution of 6060*1080 pixels, the text in the window is mixed with the settings buttons. In this form, they are difficult to use. Is there a way to fix this? Thank you.
![2023-09-29_22-28-44](https://github.com/Courseplay/Courseplay_FS22/assets/143211832/50b5fc4d-9982-4a17-83d9-d407f0b2fb66)

**Comment by Tensuko** [comment]
Sry but there is not much we can do about that.
You can try to play arround with the ingame HUD scaling, but not sure if this will have the effect you hope for.

**Reply by BanShieRU** [reply]
Very sad, but nevertheless, thanks a lot for the answer.

**Reply by Tensuko** [reply]
You could try out this Version, it may helps.
[FS22_Courseplay.zip](https://github.com/Courseplay/Courseplay_FS22/files/14618995/FS22_Courseplay.zip)

**Comment by tn4799** [comment]
Would it be fine for you if the text is cut off after a certain length to prevent the button overlap?
